task_manager/templates/index.html

```diff
@@ -1,80 +1,16 @@
-<!doctype html>
-<html lang="ru-ru">
-  <head>
-    <meta
-      name="viewport"
-      content="width=device-width, initial-scale=1, shrink-to-fit=no"
-    />
-    <title>Менеджер задач Hexlet</title>
-    <link
-      crossorigin="anonymous"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/css/bootstrap.min.css"
-      integrity="sha384-rbsA2VBKQhggwzxH7pPCaAqO46MgnOM80zW1RWuH61DGLwZJEdK2Kadq2F9CUG65"
-      rel="stylesheet"
-    />
-    <script
-      crossorigin="anonymous"
-      integrity="sha384-kenU1KFdBIe4zVF0s0G1M5b4hcpxyD9F7jL+jjXkk+Q2h455rYXK/7HAuoJl+0I4"
-      src="https://cdn.jsdelivr.net/npm/bootstrap@5.2.3/dist/js/bootstrap.bundle.min.js"
-    ></script>
-  </head>
-  <body class="d-flex flex-column min-vh-100">
-    <nav class="navbar navbar-expand-lg navbar-light bg-light ms-3">
-      <a class="navbar-brand" href="/">Менеджер задач</a>
-      <button
-        class="navbar-toggler"
-        data-bs-toggle="collapse"
-        data-bs-target="#navbarToggleExternalContent"
+{% extends "base.html" %} {% block content %} {% load i18n %}
```

Comment:
is it necessary to load i18n two times? you load it in `base.html`

Reply:
yes, it is - see the [docs](https://docs.djangoproject.com/en/4.1/topics/i18n/translation/#internationalization-in-template-code)